pymeasure/instruments/advantest/advantestR624X.py

```diff
@@ -621,18 +623,22 @@ def append_sequence_command(self, command):
 
     def init_sequence(self):
         """ This function starts the redirection of :meth:`~.write`
-        to :meth:`~.store_sequence_command`.
+        to :meth:`~.store_sequence_command` to automatically create a sequence program.
```

Comment:
I think (you should check though), that the following is sufficient as `meth` looks in the methods of the active class:
```suggestion
        to :meth:`store_sequence_command` to automatically create a sequence program.
```